code/crates/starknet/test/src/lib.rs

```diff
@@ -189,7 +189,10 @@ impl TestRunner {
                     .unwrap(),
             },
             runtime: RuntimeConfig::single_threaded(),
-            test: TestConfig::default(),
+            test: TestConfig {
+                stable_block_times: true,
```

Comment:
If this is set to false, which tests are failing and how? Is there a CI failure that shows this? I looked at the failures in 726b43472c958cae6bc55c5d4b54a88cf0ea0e5c but I think may have been because of the mempool bcast.
```suggestion
                stable_block_times: false,
```

Reply:
IIRC it's mostly the `::full_node` tests which are failing, the nodes apparently don't have time to catch up via sync. We can try to fix this, but I wanted to get this merged early to unblock @nenadmilosevic95.

Reply:
oh we should fix this but yes let's do it in a separate PR then.